quiz.py

```diff
@@ -0,0 +1,63 @@
+def startQuiz():
+    while (True):
+        print("Start Quiz")
+        num_questions = howManyQuestions("samplequiz.txt") #count number of quesitons in the file.
+        displayQuestion("samplequiz.txt") #display the question.
```

Comment:
I would prefer to  have the questions appear one at a time, with the user being prompted for an answer after each question shows. We can discuss this as a group later and see what other people prefer.